googlecompute/ubuntu_images.json

```diff
@@ -50,6 +50,12 @@
         "sudo apt-get install -y -qq git vim build-essential zip"
       ]
     },
+    {
+      "type": "shell",
+      "inline": [
+        "cp /usr/share/unattended-upgrades/20auto-upgrades-disabled /etc/apt/apt.conf.d/"
```

Comment:
Is it better then `sudo apt-get remove unattended-upgrades`?

Reply:
This seems to be a less intrusive option, especially considering this file actually exists int the first place.